js/pop.js

```diff
@@ -0,0 +1,114 @@
+const container = document.getElementById('modal');
+const workContainer = document.querySelector('.work-container');
+const objs = [
+  {
+    topic: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    img: 'image-resources/Snapshoot Portfolio3.jpg',
+    languages: ['html', 'css', 'javascript', 'Ruby on Rails'],
+    tags: [' CANOPY • BACK END DEV • 2015'],
+  },
+  {
+    topic: 'Multi Posting',
+    description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    img: 'image-resources/Snapshoot Portfolio2.jpg',
+    languages: ['html', 'css', 'javascript', 'Ruby on Rails'],
+    tags: [' UBER • LEADER BOARD 2015 '],
+  },
+  {
+    topic: 'Facebook 360',
+    description: "Exploring the future of media in Facebook's first Virtual Reality app a place to discover and enjoy 360 photos and videos on Gear VR.",
+    img: 'image-resources/Portfolio.png',
+    languages: ['html', 'css', 'javascript', 'Ruby on Rails'],
+    tags: [' FACEBOOK •Full STACK DEV • 2015'],
+  },
+  {
+    topic: 'Uber Navigation',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    img: 'image-resources/Snapshoot Portfolio3.jpg',
+    languages: ['html', 'css', 'javascript', 'Ruby on Rails'],
+    tags: ['CANOPY • UBER LEADER BOARD • 2015'],
+  },
+];
+
+for (let i = 0; i < objs.length; i += 1) {
+  workContainer.innerHTML += `<section class="work-section">
+<div class="work-section1">
+    <img src="${objs[i].img}" alt="Snapshoot image">
+</div>
+<div class="work-section2">
+
+    <div class="work-topic">
+        <h2>${objs[i].topic}</h2>
+    </div>
+    <div class="work-details">
+        <ul>
+            <li>${objs[i].tags}</li>
+        </ul>
+    </div>
+    <div class="work-paragraph">
+        <p>${objs[i].description}</div>
+    <div class="work-langs">
+         <ul>${objs[i].languages.map((m) => `<li><a href="#">${m}</a></li>`).join('')}
```

Comment:
**[REQUIRED BUT OPTIONAL]**

*I'm marking this optional but you should implement it before you merge to improve your design on the mobile version. Since you've implemented all the changes requested by the reviewers. I see your hard work* :pray: 




- Hey, your work section is great, however, the `Ruby on rails` list items on the mobile version is making your design not consistent as it is meant to have three list items in the mobile version but four in the desktop version. 
- You can choose to hide the `Ruby on rails` for the mobile version and then display it for the mobile version.

*your design now on mobile version*
![Screenshot from 2023-03-09 08-59-47](https://user-images.githubusercontent.com/59397159/223957879-174af2f5-4a6b-4dfe-b157-7f6ffecf8670.png)

*Figma design*
![Screenshot from 2023-03-09 09-00-37](https://user-images.githubusercontent.com/59397159/223958023-81f58a74-d60a-48a2-af36-627d0da0520b.png)